packages/lmi/src/lmi/llms.py

```diff
@@ -246,6 +246,19 @@ async def call(  # noqa: C901, PLR0915
         if n < 1:
             raise ValueError("Number of completions (n) must be >= 1.")
 
+        # NOTE: Checking if the model is a deepseek-r1-like model so that we can pass include_reasoning=True
+        # This hard-coded check will be removed once we have a better way to check if the model supports reasoning
+        # See: https://github.com/BerriAI/litellm/issues/8765
+        # Only reasoning with deepseek-r1 model is supported so far
+        if "deepseek" in self.name:
```

Comment:
Let's use `litellm.get_llm_provider` paired with `litellm.LlmProviders` instead of string literals